lib/matplotlib/backends/backend_svg.py

```diff
@@ -459,10 +459,7 @@ def _make_id(self, type, content):
         return '%s%s' % (type, m.hexdigest()[:10])
 
     def _make_flip_transform(self, transform):
-        return (transform +
-                Affine2D()
-                .scale(1.0, -1.0)
-                .translate(0.0, self.height))
+        return transform + Affine2D().scale(1, -1).translate(0, self.height)
```

Comment:
The multi-line version is arguably  more readable but not going to block over that.

Reply:
+/-0